mne_bids/copyfiles.py

```diff
@@ -339,6 +340,94 @@ def copyfile_brainvision(vhdr_src, vhdr_dest, anonymize=None, verbose=False):
             print('Anonymized all dates in VHDR and VMRK.')
 
 
+def copyfile_edf(src, dest, anonymize=None):
+    """Copy an EDF, EDF+, or BDF file to a new location, optionally anonymizing
+    participant/patient information.
```

Comment:
according to numpydoc, the first line of a docstr must be a single sentence in imperative voice.

```suggestion
    """Copy an EDF, EDF+, or BDF file to a new location, optionally anonymize.
```